### PR TITLE
[docs][user-authz] Documentation updates

### PR DESCRIPTION
**Pull request by z9r5**
## Description
Documentation updates.

## Changelog entries
```changes
section: user-authz
type: fix
summary: Documentation updates
impact_level: low
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

File: modules/140-user-authz/docs/USAGE_RU.md [hunk 1]
```diff
@@ -314,19 +319,31 @@ apiVersion: rbac.authorization.k8s.io/v1
...
 metadata:
   annotations:
-    user-authz.deckhouse.io/access-level: PrivilegedUser
-  name: d8-mymodule-ns:privileged-user
+    user-authz.deckhouse.io/access-level: Editor
+  name: myapp-ns:privileged-user
 rules:
 - apiGroups:
-  - mymodule.io
+  - networking.istio.io
```
[the remark this answers is not in the document]
Comment (z9r5) [reply modules/140-user-authz/docs/USAGE_RU.md]: Made the changes.

File: modules/140-user-authz/docs/USAGE_RU.md [hunk 2]
```diff
@@ -314,19 +319,31 @@ apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
   annotations:
-    user-authz.deckhouse.io/access-level: PrivilegedUser
-  name: d8-mymodule-ns:privileged-user
+    user-authz.deckhouse.io/access-level: Editor
+  name: myapp-ns:privileged-user
```
[the remark this answers is not in the document]
Comment (z9r5) [reply modules/140-user-authz/docs/USAGE_RU.md]: Made the changes.

**Review by nabokihms — APPROVED** [review]

**Comment by z9r5** [comment]
Cherry-picked into release-1.33

**Pull request merged**